package.json

```diff
@@ -44,7 +44,8 @@
     "@babel/preset-env": "7.16.11",
     "@babel/preset-react": "7.16.7",
     "@babel/preset-typescript": "7.16.7",
-    "@material-ui/core": "^4.12.3",
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
```

Comment:
In the [upgrade guide](https://mui.com/material-ui/migration/migration-v4/#introduction), inside introductory section, they say:

> One of the biggest changes in v5 is the replacement of JSS for [Emotion](https://emotion.sh/docs/introduction) as a default styling solution.

But, we are already using `styled-components`. In the [peer dependency section](https://mui.com/material-ui/migration/migration-v4/#peer-dependencies), they say you should use `@mui/styled-engine-sc` instead of Emotion packages.

I see you installed Emotion packages here (devDependency) and `@mui/styled-engine-sc` as a regular dependency.